scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -79,6 +79,51 @@ SOP-8_3.76x4.96mm_P1.27mm:
   num_pins_x: 0
   num_pins_y: 4
 
+SOP-8_3.9x4.9mm_P1.27mm:
+  size_source: 'http://www.macronix.com/Lists/Datasheet/Attachments/7534/MX25R3235F,%20Wide%20Range,%2032Mb,%20v1.6.pdf'
+  body_size_x:
+    minimum: 3.80
+    maximum: 4.00
+  body_size_y:
+    minimum: 4.77
+    maximum: 5.03
+  overall_height:
+    maximum: 1.75
+  overall_size_x:
+    minimum: 5.80
+    maximum: 6.20
+  lead_width:
+    minimum: 0.36
+    maximum: 0.51
+  lead_len:
+    minimum: 0.46
+    maximum: 0.86
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 4
+
+SOP-8_5.28x5.23mm_P1.27mm:
+  size_source: 'http://www.macronix.com/Lists/Datasheet/Attachments/7534/MX25R3235F,%20Wide%20Range,%2032Mb,%20v1.6.pdf'
```

Comment:
add #page=80 to the link to point directly to the correct datasheet page.

Reply:
Done